src/IW/Server/Types.hs

```diff
@@ -0,0 +1,15 @@
+-- | This module introduce aliases to use for @servant-generic@ types and functions writing.
+
+module IW.Server.Types
+       ( AppServer
+       , ToApi
+       ) where
+
+import Servant.API.Generic (ToServantApi)
+import Servant.Server.Generic (AsServerT)
+
+import IW.App (App)
+
+
+type AppServer = AsServerT App
+type ToApi (site :: * -> *) = ToServantApi site
```

Comment:
Let's write here `Type -> Type` as well instead of `* -> *` for consistency.

Reply:
Oh yeah got it.